### PR TITLE
API1600 support for Volume

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[API1600 support for Volume]

### Issues Resolved
[]

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: spec/unit/resource/api1600/synergy/volume_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1200::Synergy::Volume do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API1200::C7000::Volume' do
+    expect(described_class).to be < OneviewSDK::API1200::C7000::Volume
```
Comment (chebroluharika) [inline spec/unit/resource/api1600/synergy/volume_spec.rb:7]: I think you should inherit from API1600::C7000::Volume. If so, please update description also accordingly.

**Comment by nabhajit-ray** [comment]
> needs to correct the spec/unit/resource/api1600/synergy/volume_spec.rb file.

done

File: spec/unit/resource/api1600/synergy/volume_spec.rb (hunk 1 above)
Comment (nabhajit-ray) [reply spec/unit/resource/api1600/synergy/volume_spec.rb]: done

File: spec/unit/resource/api800/synergy/volume_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API800::Synergy::Volume do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API800::C7000::Volume' do
+    expect(described_class).to be < OneviewSDK::API800::C7000::Volume
```
Comment (VenkateshRavula) [inline spec/unit/resource/api800/synergy/volume_spec.rb:18]: the lib file inherits from API600::Synergy.. Please check

Comment (VenkateshRavula) [inline spec/unit/resource/api800/synergy/volume_spec.rb:18]: the lib file inherits from API600::Synergy.. Please check

**Pull request merged**